### ISSUE TITLE
Doesn't build on Ubuntu 18.04, it needs CMake 3.13

### ISSUE THREAD

**Issue by pts** [darlinghq/darling#1364]
**Build Log**
What is the build error?
```
$ cmake -DFULL_BUILD=OFF ..
Python 2 not available; bytecode compilation is disabled
-- Could NOT find Vulkan (missing: Vulkan_LIBRARY Vulkan_INCLUDE_DIR) 
Did not find required libraries (Vulkan and LLVM); building without Metal support
-- Found dsymutil: /usr/bin/llvm-dsymutil-6.0
-- Compiler include path detected as /usr/lib/llvm-6.0/lib/clang/6.0.0/include/
CMake Error at src/external/darlingserver/CMakeLists.txt:3 (cmake_minimum_required):
  CMake 3.13 or higher is required.  You are running version 3.10.2


-- Configuring incomplete, errors occurred!
See also "/home/pts/Downloads/darling/darling/build/CMakeFiles/CMakeOutput.log".
```
https://docs.darlinghq.org/build-instructions.html explicitly says mentions that Ubuntu 18.04.

Also, the error message `Python 2 not available; bytecode compilation is disabled` is incorrect. Python 2 is indeed installed, it's the `python` command from the corresponding Ubuntu 18.04 package.

Please note that I'm a new user of Darling. I'm running Ubuntu 18.04. I'd like to run text-mode macOS i386 programs with Darling on Linux. I don't care about building Darling from sources. I just want to run it. Is there a recent Docker image or a binary release available, so that I don't have to bother with compilation?

**System Information**
Ubuntu 18.04

| Software | Version |
| --- | --- |
| Clang | 6.0.0-1ubuntu2 |
| CMake | 3.13 |
| Linux Kernel | 5.4.0-124-generic |
| Darling | 557e7e9dece394a3f623825679474457e5b64fd0 |

**Comment by facekapow** [comment]
> https://docs.darlinghq.org/build-instructions.html explicitly mentions that Ubuntu 18.04, so I'm expecting that Darling should build there.

Yeah, that message hasn't been updated in a bit (we should probably remove "18.04" from that header); we've since updated our CMake requirements to 3.13, since we use some features it provides. However, newer CMake versions are still available on 18.04 via Kitware's official repo (https://apt.kitware.com/).

> Is there a recent Docker image

Yes, but it's quite old (from when we still had a kernel module).

> or a binary release available

Yes, automatically built by the CIs, but only for 20.04.

> Also, the error message `Python 2 not available; bytecode compilation is disabled` is incorrect. Python 2 is indeed installed, it's the `python` command from the corresponding Ubuntu 18.04 package.

Yeah, I'm not sure why it says that; it's certainly supposed to search for both `python` and `python2`.

**Comment by neomafo88** [comment]
https://askubuntu.com/questions/355565#865294
